omnibus-arbitrum-sepolia-octopus.toml

```diff
@@ -0,0 +1,185 @@
+name = "synthetix-omnibus"
```

Comment:
Please make sure the congfig follows `omnibus-arbitrum-sepolia.toml` structure, order and conventions, otherwise merging them together will be much more difficult